### PR TITLE
create wmstats link according to request origin

### LINKED ISSUE #6034: Clean up WQ and other caches created by Request.py in Service package

### PR DESCRIPTION
**Pull request by ticoann**
fixes #6034 - tested in my vm

### CODE REVIEW

**Comment by ticoann** [comment]
@amaltaro, Alan, please review

File: src/couchapps/WMStats/_attachments/js/minified/global.min.js [hunk 1]
```diff
@@ -66,6 +67,16 @@ WMStats.Globals = function($){
     
     function getReqDetailPrefix () {
         if (_dbVariants[dbname] == "tier1") {
```
Comment (amaltaro) [inline src/couchapps/WMStats/_attachments/js/minified/global.min.js]: What is this tier1 dbname? Is it still in use? The same for the analysis one below.

**Comment by amaltaro** [comment]
Patch looks good to me. Thanks!

File: src/couchapps/WMStats/_attachments/js/minified/global.min.js (hunk 1 above)
Comment (ticoann) [inline src/couchapps/WMStats/_attachments/js/minified/global.min.js]: I am not sure analysis one is used (may be not)  but we have to distinguish between tier0 and production.

**Pull request merged**